src/browser/provider/built-in/chrome/cdp.js

```diff
@@ -0,0 +1,110 @@
+import remoteChrome from 'chrome-remote-interface';
+import fs from 'fs';
+import Promise from 'pinkie';
+import pify from 'pify';
+
+
+const fsWriteFile = pify(fs.writeFile, Promise);
+
```

Comment:
Use `promisify` here from `src/utils/promisify.js` and below

Also we can move all promisified standard function to the separate file.
What do you think?